### PR TITLE
Fix class imports

### PR DESCRIPTION
**Pull request by Soullivaneuh**

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
ping @henrikbjorn.

File: Twig/GravatarExtension.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 namespace Ornicar\GravatarBundle\Twig;
 
+use Ornicar\GravatarBundle\Templating\Helper\GravatarHelper;
```
Comment (henrikbjorn) [inline Twig/GravatarExtension.php]: We dont need this one, the property phpdoc should be removed or change to the interface aswell.

Comment (Soullivaneuh) [inline Twig/GravatarExtension.php]: Indeed.

**Comment by Soullivaneuh** [comment]
@henrikbjorn needed changes done.

**Comment by henrikbjorn** [comment]
thanks:)

**Comment by Soullivaneuh** [comment]
I think you click on the wrong button... :-)

**Comment by henrikbjorn** [comment]
Woups.

**Comment by Soullivaneuh** [comment]
Better. :+1:

**Pull request merged**